frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/mirroring-card/mirroring-card.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { K8sResourceKind } from '@console/internal/module/k8s/index';
+import { FirehoseResource } from '@console/internal/components/utils/index';
+import { referenceForModel } from '@console/internal/module/k8s/k8s';
+
+import { OCSServiceModel } from '../../../../models';
+import { CEPH_STORAGE_NAMESPACE } from '../../../../constants';
+import '../../common/storage-efficiency/storage-efficiency-card.scss';
+
+export const MirroringItemBody: React.FC<MirroringItemBodyProps> = React.memo(
+  ({ stats, title, isLoaded, error, getStats }) => {
+    const { t } = useTranslation();
+
+    let status: React.ReactElement;
+
+    if (!isLoaded) {
+      status = <div className="skeleton-text ceph-storage-efficiency-card__item-body--loading" />;
+    } else if (error || !stats) {
+      status = (
+        <span className="co-dashboard-text--small text-muted">
+          {t('ceph-storage-plugin~Disabled')}
```

Comment:
I am creating a similar react element in my dashboard PR:
https://github.com/openshift/console/pull/9423/files#diff-acc71ab7bfff17bdf71b4d1fccdfaca70685641d5fec824a75a9f9ecc5b80d7bR5

https://github.com/openshift/console/pull/9423/files#diff-aa479e95de24a9a068e30fa9ab25f8e33574f37c8ee80160eaa3a7d5b627b0d8R1

Please create the same folder and file in this PR and use it here.

Reply:
ack...